pkgs/development/libraries/wxwidgets/wxGTK32.nix

```diff
@@ -0,0 +1,147 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchurl
+, gnome2
+, gst_all_1
+, gtk2
+, gtk3
+, libGL
+, libGLU
+, libSM
+, libXinerama
+, libXtst
+, libXxf86vm
+, pkg-config
+, xorgproto
+, compat28 ? false
+, compat30 ? true
+, unicode ? true
```

Comment:
Do we need those options if we are honest to us? If not we could just drop them.

Reply:
I don't know. It's coming from other versions.

Reply:
We might need `compat30` in the future when [building wxQt](https://docs.wxwidgets.org/trunk/plat_qt_install.html) (which is not in nixpkgs yet, though).